app/repositories/onlinetesting/Phase2TestRepository.scala

```diff
@@ -83,11 +83,36 @@ class Phase2TestMongoRepository(dateTime: DateTimeFactory)(implicit mongo: () =>
   }
 
   override def nextApplicationsReadyForOnlineTesting: Future[List[OnlineTestApplication]] = {
-    val query = BSONDocument(
-      "applicationStatus" -> ApplicationStatus.PHASE1_TESTS_PASSED,
-      "assistance-details.needsSupportForOnlineAssessment" -> false,
-      s"progress-status.${PHASE1_TESTS_PASSED}" -> true
-    )
+    // new
+    val query =
+      BSONDocument("$and" -> BSONArray(
+        BSONDocument("applicationStatus" -> ApplicationStatus.PHASE1_TESTS_PASSED),
+        BSONDocument(s"progress-status.${PHASE1_TESTS_PASSED}" -> true),
+        BSONDocument("$or" -> BSONArray(
+          BSONDocument("$and" -> BSONArray(
+            BSONDocument("assistance-details.needsSupportForOnlineAssessment" -> false),
+            BSONDocument("assistance-details.needsSupportAtVenue" -> false),
+            BSONDocument("assistance-details.guaranteedInterview" -> BSONDocument("$ne" -> true)))),
+          BSONDocument("$and" -> BSONArray(
+            BSONDocument("$or" -> BSONArray(
+              BSONDocument("assistance-details.guaranteedInterview" -> true),
```

Comment:
@ollyw The existing code should cover that: we are not inviting those who have invigilated etray adjustment.